src/css/hero.css

```diff
@@ -1,159 +1,504 @@
+}
+
+@media (-webkit-min-device-pixel-ratio: 2),
+  (min-resolution: 192dpi),
+  (min-resolution: 2dppx) {
+  .hero-container {
+    background-image: url(../img/mobilestrawberries@2x.png),
+      url(../img/mobilebananas@2x.png);
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .hero-container {
+    background-image: url(../img/tabletstrawberries.png),
+      url(../img/tabletbananas.png);
+    background-size: 108px 108px, 132px 90px;
+    background-position: top 54px right -5px, top 169px left -5px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .hero-container {
+      background-image: url(../img/tabletstrawberries@2x.png),
+        url(../img/tabletbananas@2x.png);
+    }
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  .hero-container {
+    background-image: url(../img/desktopstrawberries.png),
+      url(../img/desktopbananas.png);
+    background-size: 120px 120px, 183px 125px;
+    background-position: top 39px right 191px, top 191px left 181px;
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .hero-container {
+      background-image: url(../img/desktopstrawberries@2x.png),
+        url(../img/desktopbananas@2x.png);
+    }
+  }
+}
+.header-primary {
+  padding-top: 56px;
+  margin-bottom: 386px;
+  font-weight: 500;
+  font-size: 28px;
+  color: #ffffff;
+  line-height: 1;
+  letter-spacing: -0.04em;
+  text-align: center;
+  text-transform: uppercase;
+  text-shadow: 0px 4px 70px rgba(0, 0, 0, 0.1);
+}
+
+@media only screen and (min-width: 768px) {
   .header-primary {
-    font-family: 'DM Sans', sans-serif;
-    font-weight: 500;
-    font-size: 64px;
+    padding-top: 64px;
+    margin-bottom: 513px;
+    font-size: 50px;
+    line-height: 1.08;
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  .header-primary {
+    margin-bottom: 483px;
     line-height: 1;
-    letter-spacing: -0.04em;
-    text-transform: uppercase;
-    text-align: center;
-    color: #fff;
-    text-shadow: 0 4px 70px 0 rgba(0, 0, 0, 0.1);
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 930px;
+    font-size: 64px;
   }
 }
 
-@media screen and (min-width: 1280px) {
-  .hero-strawberries {
-    width: 120px;
-    height: 120px;
-    filter: blur(8px);
+.hero-button {
+  position: relative;
+  margin: 0 auto;
+  margin-bottom: 44px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 80px;
+  height: 80px;
+  background-color: #ffffff;
+  cursor: pointer;
+  border: none;
+  border-radius: 500px;
+  color: #ae1332;
+  transition: color 400ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.hero-button-text {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 1;
+  letter-spacing: -0.04em;
+  text-transform: uppercase;
+  text-align: center;
```

Comment:
dodaj proszę:

margin: auto;

Reply:
to nic nie zmieni. tekst w przycisku jest idealnie na środku bo wypełnia hero-button i ustawiony jest poprzez text-align: center